src/Controller/ApiController.php

```diff
@@ -100,53 +108,93 @@ public function viewPortfolioList(Request $request, int $page, int $limit, array
             $page = 1;
         }
 
-        $cats = $request->query->all('cats');
+        $pid = $request->query->all('pid');
+        $lang = $request->query->get('lang') ?: $GLOBALS['TL_LANGUAGE'];
 
         $offset = ($page - 1) * $limit;
-        if (!is_iterable($cats)) {
-            return new JsonResponse('{"error":"Give at least on category : ?cats[]=1&cats[]=2"}', Response::HTTP_NOT_ACCEPTABLE, [], true);
+        if (!is_iterable($pid)) {
+            return new JsonResponse('{"error":"Give at least on category : ?pid[]=1&pid[]=2"}', Response::HTTP_NOT_ACCEPTABLE, [], true);
```

Comment:
`at least one`

Reply:
Fixed in 5ff0bedc0429915c064e1a13e56e562c45b50ff2